addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -236,7 +240,7 @@ private synchronized void refreshData() {
 		try {
 			refreshDeviceData();
 			if (connectionEstablished) {
-				updateStatus(ThingStatus.ONLINE);
+				if (!getThing().getStatus().equals(ThingStatus.ONLINE)) updateStatus(ThingStatus.ONLINE);
```

Comment:
> Also as the status update message come as INFO, I found it bit polluting the logs.

This should not be a topic that bindings try to address themselves, see https://github.com/eclipse/smarthome/pull/430.